c7n/resources/mq.py

```diff
@@ -82,6 +83,57 @@ def get_dimensions(self, resource):
         return [{'Name': self.model.dimension,
                  'Value': "{}-1".format(resource['BrokerName'])}]
 
+@MessageBroker.filter_registry.register('vpc')
+class VpcFilter(VpcFilter):
+
```

Comment:
It might be helpful to add a class-level docstring that gives a summary of the filter and an example of what a policy using this filter will look like. See the `MarkForOpMessageBroker` action in this file for an example.

https://realpython.com/documenting-python-code/ discusses docstrings and type hinting in more detail if you're interested!